digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/util/DataFixture.scala

```diff
@@ -1672,6 +1677,26 @@ object DataFixture {
     println("\n")
   }
 
+  def loadMunicipalitiesVerificationInfo(): Unit = {
+    println("\nRefreshing information on municipality verification")
+    println(DateTime.now())
+
+    //Get All Municipalities
+    val municipalities: Seq[Int] = OracleDatabase.withDynSession { Queries.getMunicipalities  }
+    OracleDatabase.withDynTransaction {
```

Comment:
Since we will use municipalities variable inside of DynTransaction, why we cannot perform the query to get municipalities inside them?